Sources/App/Logger/Logger.swift

```diff
@@ -1,7 +1,8 @@
+import Vapor
 import Bot
 
-public final class PrintLogger: LoggerProtocol {
+public final class PrintLogger: LoggerProtocol, Service {
     public func log(_ message: String) {
-        print("[WALL-E] \(message)")
+        print("| \(Date()) | [WALL-E] \(message)")
```

Comment:
```suggestion
        print("\(Date()) | [WALL-E] \(message)")
```

Please ignore if this prefix has some meanings.